extensions/pyRevitTools.extension/pyRevit.tab/Project.panel/Preflight Checks.pushbutton/config.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: UTF-8 -*-
+
+from os.path import dirname, abspath, join
+from pyrevit import script
+from pyrevit.forms import alert, pick_file, pick_folder
+from csv import reader
+
+script.get_output().close_others()

```

Comment:
Is this really needed? this doesn't open any output window after all...

Reply:
Old habit, I run tools so many times that I get dozens of output windows. This allows me to start with a clean slate.